AusTraits_tutorial.qmd

```diff
@@ -2,104 +2,116 @@
 
 ## Introduction
 
-With more than 1.8 million data records, AusTraits is Australia's [largest plant trait database](austraits_database.html#plant_database), created using the [`traits.build`] package(https://github.com/traitecoevo/traits.build)
+With more than 1.8 million data records, AusTraits is Australia's [largest plant trait database](austraits_database.html#plant_database), created using the [`{traits.build}` R package](https://github.com/traitecoevo/traits.build)
 
+This tutorial introduces:
 
-The following tutorial introduces:
+-   [the database structure](#database_structure)
 
--   [the database structure](#database_structure)  
--   
--   additional examples of analyses using the database  
--   
+-   [`{austraits}` R package functions](#austraits_functions)
 
+-   additional [examples of analyses](#sample_analyses) using the database
 
-To access the Github repositories for individual components, see: 
+To access more information about `traits.build` see [`traits.build-book`](https://traitecoevo.github.io/traits.build-book/)  
+
+
+Or you can visit the Github repositories for individual packages/data repos:
 
 -   the database structure, [`traits.build`](https://github.com/traitecoevo/traits.build)
 
 -   the database contents, [`austraits.build`](https://github.com/traitecoevo/austraits.build)
 
 -   an R package for exploring and wrangling the data, [`austraits`](https://github.com/traitecoevo/austraits)
 
-
-
 ## Download AusTraits data
 
-Before you begin, download (or build) the latest AusTraits data, by one of the methods described [here](austraits_database.html#access_data)
+Before you begin, source essential packages and functions.
 
-```{r, eval = FALSE, message = FALSE}
+```{r, eval = TRUE, message = FALSE}
 library(readr)
 library(tidyr)
 library(dplyr)
 library(stringr)
-library(austraits)
-library(APCalign)
+
+library(austraits)    # functions for exploring a traits.build database, available at Github repo
+library(traits.build) # additional functions for exploring a traits.build database, available at Github repo
+library(APCalign)     # Australian plant taxon alignment function, available on CRAN 
 
 source("data/extra_functions.R")
+```
+
+Then download (or build) the latest AusTraits data, by one of the methods described [here](austraits_database.html#access_data).
 
-austraits <- read_rds("data/austraits_5.Rds")
+This tutorial uses the most recent AusTraits release, version `r austraits::get_versions()[[2]][1]`
```

Comment:
Should this be [[3]]?

Also up to you but could swap to tidyverse version of code for readability:
```austraits::get_versions() |> dplyr::pull("version") |> dplyr::first()```

Reply:
Good idea - but the [[2]] is correct, because I'm loading from the doi. Partially so people see that there is a doi for the version